### DISCUSSION TITLE
Download problem

### DISCUSSION THREAD

**Discussion by SDGArmenia** [open-sdg/open-sdg#2123] [Q&A] [answered]
Hi all,
We recently upgraded our platforms to 2.3.0. 
We have a problem with data editing. 
"Download data" button doesn't functions.
https://armstat.github.io/sdg-site-armenia/17-1-1/data/
Can anyone help to solve the problem.    
Тhank you in advance.

**Comment by brockfanning** [comment, marked as answer]
@SDGArmenia Thank you for pointing out this bug. Can you please try upgrading to version of 2.3.3 of open-sdg/open-sdg in your _config.yml file?

`remote_theme: open-sdg/open-sdg@2.3.3`

**Reply by SDGArmenia** [reply]
Thank you very much. We have updated open-sdg/open-sdg to version 2.3.3. Now the  button Download works
